core/src/main/java/com/sekwah/advancedportals/core/tags/TriggerBlockTag.java

```diff
@@ -48,4 +55,29 @@ public List<String> autoComplete(String argData) {
 
         return triggerBlocks;
     }
+
+    @Override
+    public boolean created(TagTarget target, PlayerContainer player, String[] argData) {
+        for (int i = 0; i < argData.length; i++) {
+            String material = argData[i].toUpperCase(Locale.ROOT);
```

Comment:
dont automatically convert to uppercase, one issue is that when we add the forge versions their materials are in lower case. Preferably, add a method to the server container to try to match the material and return the matched name. That way we can make it per implementation based.